docker/src/main/java/org/jclouds/docker/domain/NetworkSettings.java

```diff
@@ -59,6 +59,77 @@ public static Details create(String endpointId, String gateway, String ipAddress
          return new AutoValue_NetworkSettings_Details(endpointId, gateway, ipAddress, ipPrefixLen, ipv6Gateway, globalIPv6Address, globalIPv6PrefixLen,
                  macAddress);
       }
+      
+      public static Builder builder() {
```

Comment:
as you are adding the builders, can you please use [this](https://github.com/google/auto/blob/master/value/userguide/builders.md) ?